packages/library/src/base/component.ts

```diff
@@ -35,7 +35,7 @@ enum PrivateEventName {
   endUncontrolled = 'end:uncontrolled',
 }
 
-const EventName = { ...PublicEventName, ...PrivateEventName }
+export const EventName = { ...PublicEventName, ...PrivateEventName }
```

Comment:
Note to self: This is an unused export, I think (that I introduced)

Reply:
Correct